src/platform/src/Bridge/Groq/Llama.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Bridge\Groq;
+
+use Symfony\AI\Platform\Capability;
+use Symfony\AI\Platform\Model;
+
+/**
+ * @author Christopher Hertel <mail@christopher-hertel.de>
+ * @author Dave Hulbert <dave1010@gmail.com>
+ */
+class Llama extends Model
```

Comment:
could be something more neutral like
```suggestion
class Model extends BaseModel
```

since this is not only Llama it's a bit confusing - it's rather a collection of models available on Groq